athena-federation-integ-test/pom.xml

```diff
@@ -198,7 +198,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.14.0</version>
+            <version>3.12.0</version>
```

Comment:
why are we downgrading?

Reply:
It conflicts with internal Engine version; it has to be the same version or lower.